src/components/timer/rangebar.js

```diff
@@ -0,0 +1,19 @@
+import StyledInputRange from './styledInputRange';
+
+export default function RangeBar({value, setValue, name}) {
+	return (
+		<>
+			<StyledInputRange
+				type="range"
+				id={name}
+				name={name}
+				min="00"
+				max="59"
+				value={value}
+				onChange={event => {
+					setValue(Number(event.target.value));
+				}}
```

Comment:
I would recommend to not pass the `set` function from a state directly to a component but pass a callback function instead:
```suggestion
export default function RangeBar({value, onChange, name}) {
	return (
		<>
			<StyledInputRange
				type="range"
				id={name}
				name={name}
				min="00"
				max="59"
				value={value}
				onChange={event => {
					onChange(Number(event.target.value));
				}}
```

Reply:
ich benutze den onChange prob ja davor, ich kann das leider nicht ändern oder sehe ich das falsch?